src/main/java/core/basesyntax/PasswordValidationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class PasswordValidationException extends RuntimeException {
```

Comment:
What is the difference between RuntimeException and Exception?

Reply:
done